Makefile

```diff
@@ -132,10 +132,13 @@ clean-aci-e2e: ## Make sure no ACI tests are currently runnnig in the CI when in
 	@ az group list | jq '.[].name' | grep -i E2E-Test
 	az group list | jq '.[].name' | grep -i E2E-Test | xargs -n1 az group delete -y --no-wait -g
 
+docs:
```

Comment:
already there: https://github.com/docker/compose-cli/blob/main/builder.Makefile#L100

Reply:
could maybe be containerized from this Makefile?

Reply:
/rebase


just testing the rebase trigger